package.json

```diff
@@ -4,10 +4,15 @@
   "description": "A declarative Svelte routing library with SSR support",
   "main": "src/index.js",
   "svelte": "src/index.js",
+  "types": "types/index.d.ts",
   "author": "Emil Tholin @EmilTholin",
   "license": "MIT",
   "repository": "EmilTholin/svelte-routing",
   "peerDependencies": {
     "svelte": "^3.20.x"
+  },
+  "devDependencies": {
+    "svelte": "^3.31.1",
+    "svelte2tsx": "^0.1.157"
```

Comment:
I think this needs to be a regular dependencie. Otherwise npm would not install the types when someone installes svelte-routing...

Reply:
Indeed, my mistake, i assumed those were only development-only types